### PR TITLE
fix: retrieve not hidden items

### PR DESCRIPTION
**Pull request by juliswer**

### CODE REVIEW

File: src/items/repositories/item.repository.ts [hunk 1]
```diff
@@ -68,7 +68,7 @@ export class ItemRepository extends Repository<Item> {
 
   async getNewest(qty: number): Promise<Item[]> {
     return await this.getItemQueryBuilder()
-      .where('item.status=:status', { status: ItemStatus.Listed })
+      .where('item.status != :status', { status: ItemStatus.Hidden })
```
Comment (mmiranda-stackit) [inline src/items/repositories/item.repository.ts:72]: what about draft items ? Is that even possible ?

File: src/items/repositories/item.repository.ts [hunk 2]
```diff
@@ -130,7 +130,7 @@ export class ItemRepository extends Repository<Item> {
 
   async findAll(): Promise<Item[]> {
     return this.getItemQueryBuilder()
-      .where('item.status=:status', { status: ItemStatus.Listed })
+      .where('item.status != :status', { status: ItemStatus.Hidden })
```
Comment (mmiranda-stackit) [inline src/items/repositories/item.repository.ts:135]: Draft items ?

File: src/items/repositories/item.repository.ts [hunk 3]
```diff
@@ -422,7 +420,7 @@ export class ItemRepository extends Repository<Item> {
   private getPaginationConditions(paginationDto: ItemPaginationDto): FindOptionsWhere<Item>[] {
     const { priceFrom, priceTo, search } = paginationDto;
     const conditions = [] as FindOptionsWhere<Item>[];
-    const condition1 = { status: ItemStatus.Listed } as FindOptionsWhere<Item>;
+    const condition1 = { status: Not(ItemStatus.Hidden) } as FindOptionsWhere<Item>;
```
Comment (mmiranda-stackit) [inline src/items/repositories/item.repository.ts]: Same here

File: src/items/repositories/item.repository.ts (hunk 1 above)
Comment (juliswer) [reply src/items/repositories/item.repository.ts:72]: Good point. lemme fix it

**Review by mmiranda-stackit — APPROVED** [review]

**Pull request merged**